Detections/MultipleDataSources/ZincJan272021IOCs.yaml

```diff
@@ -76,33 +79,40 @@ query: |
   | where isnotempty(Hashes)
   | parse Hashes with * 'SHA256=' SHA256 ',' * 
   | where SHA256 in~ (SHA256Hash) 
-  | extend Account = UserName
+  | extend DataType = "Microsoft-Windows-Sysmon", Account = UserName, FileHash = Hashes
```

Comment:
This one you can use the Source and map it to Type.  I would recommend that it is clear what DataType it comes from, so a concat is probably best -  | extend Type = strcat(Type, ": ", Source)